src/main/java/frc/robot/SwerveModule.java

```diff
@@ -15,71 +15,73 @@
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
 public class SwerveModule {
-    private final CANSparkMax driveMotor;
-    private final CANSparkMax steerMotor;
+    private final CANSparkMax mDriveMotor;
+    private final CANSparkMax mSteerMotor;
     
-    private final RelativeEncoder driveEncoder;
-    private final RelativeEncoder steerEncoder;
+    private final RelativeEncoder mDriveEncoder;
+    private final RelativeEncoder mSteerEncoder;
 
-    private final PIDController steerPidController;
+    private final PIDController mSteerPidController;
 
-    private final CANCoder absoluteEncoder;
-    private final boolean absoluteEncoderReversed;
-    private final double absoluteEncoderOffsetRad;
+    private final CANCoder mAbsoluteEncoder;
+    private final boolean mAbsoluteEncoderReversed;
+    private final double mAbsoluteEncoderOffsetRad;
 
     public SwerveModule(int driveMotorId, int steerMotorId, boolean driveMotorReversed, boolean steerMotorReversed,
     int absoluteEncoderId, double absoluteEncoderOffset, boolean absoluteEncoderReversed) {
     //absolute encoders
-    this.absoluteEncoderOffsetRad = absoluteEncoderOffset;
-    this.absoluteEncoderReversed = absoluteEncoderReversed;
-    absoluteEncoder = new CANCoder(absoluteEncoderId); //default is degrees per second
-    absoluteEncoder.configFeedbackCoefficient(ModuleConstants.ABSOLUTE_ENCODER_CPM_TO_RAD, "rad", SensorTimeBase.PerSecond); //convert to radians per second
+    this.mAbsoluteEncoderOffsetRad = absoluteEncoderOffset;
+    this.mAbsoluteEncoderReversed = absoluteEncoderReversed;
```

Comment:
Not an important change, but because the variable name isn't the same as the constructor's argument name, you don't need to use `this.` anymore.

Reply:
kk. done